src/patterns/ec2-app.ts

```diff
@@ -88,9 +136,16 @@ export class GuEc2App {
     const loadBalancer = new GuApplicationLoadBalancer(scope, "LoadBalancer", {
       app,
       vpc,
-      vpcSubnets: {
-        subnets: props.publicFacing ? GuVpc.subnetsfromParameter(scope, { type: SubnetType.PUBLIC }) : privateSubnets,
-      },
+      internetFacing: true,
+      vpcSubnets: { subnets: GuVpc.subnetsfromParameter(scope, { type: SubnetType.PUBLIC, app }) },
+      securityGroup: new GuSecurityGroup(scope, "SecurityGroup", {
+        app,
+        vpc,
+        ingresses:
+          props.access?.scope === RESTRICTED
+            ? restrictedCidrRanges(props.access.cidrRanges)
+            : [{ range: Peer.anyIpv4(), port: 443, description: "Allow from anyone on port 443" }],
```

Comment:
To keep things DRY, I think we can move the ternary up and reuse the [`GuPublicInternetAccessSecurityGroup` construct](https://github.com/guardian/cdk/blob/61b0c66af7c1b0992358738d8d0effaabd8f9812/src/constructs/ec2/security-groups/base.ts#L78-L87) for the public access scope here.

Reply:
Nice, thanks. Didn't know about that one.